samples/Mvc.DynamicCorsPolicyProvider/Readme.md

```diff
@@ -0,0 +1,10 @@
+Mvc.DynamicCorsPolicyProvider Sample
+------------------------------------
+This sample demonstrates a way to dynamically load CORS policies in a running application without restarting it. 
+
+ICorsPolicyProvider is the DI injectable service through which MVC retrieves CORS policy for a request. The default CORS policy 
+provider which ships out of the box gets the policies from an in-memory configuration. Since we want to be able to 
```

Comment:
Can remove `which ships out of the box` (if it's the default, of course it's there).